src/Data/Auth/KeycloakProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Bullseye, Spinner } from "@patternfly/react-core";
+import { ReactKeycloakProvider } from "@react-keycloak/web";
+import Keycloak, { KeycloakConfig } from "keycloak-js";
+
+export interface AuthConfig extends KeycloakConfig {
+  method: "oidc";
+}
+
+export interface LocalConfig {
```

Comment:
Why is this one in the KeycloakProvider?

Reply:
That was the top place that requires it, and when I had it in AuthProvider file it created circular dependency and I didn't feel like type.ts file is required here, but I can be mistaken.

Reply:
Moving an export to another file should not be triggering circular dependency. If you have a circular dep issue, you might have forgotten to remove the import statement if you moved it to the file where it's used or adjust it.

Reply:
it does, at least madge complains when the interface is in the authprovider, app and tests works just fine

Reply:
I believe it's only used in the authProvider component, so either;
1.  don't export it and declare it as an interface inside AuthProvider.
2.  since we are using a switch case on the config.method, perhaps this distinction isn't needed anymore and we can go for a more generic type which has the different possibilities for '`method : "oidc" | "database" | "other" | undefined`

Reply:
It's reused, and in case of "oidc" the rest of the config is being passed to the keycloakProvider to create keycloak instance, I'll put it to the types.ts file to have it in "more correct" place